app/notifications/validators.py

```diff
@@ -135,6 +161,58 @@ def warn_about_daily_message_limit(service, messages_sent):
             raise LiveServiceTooManyRequestsError(service.message_limit)
 
 
+def warn_about_daily_sms_limit(service, messages_sent):
+    nearing_sms_daily_limit = messages_sent >= NEAR_DAILY_LIMIT_PERCENTAGE * service.sms_daily_limit
+    over_sms_daily_limit = messages_sent >= service.sms_daily_limit
+    current_time = datetime.utcnow().isoformat()
+    cache_expiration = int(timedelta(days=1).total_seconds())
+
+    # Send a warning when reaching 80% of the daily limit
+    if nearing_sms_daily_limit:
+        cache_key = near_sms_daily_limit_cache_key(service.id)
+        if not redis_store.get(cache_key):
+            redis_store.set(cache_key, current_time, ex=cache_expiration)
+            send_notification_to_service_users(
+                service_id=service.id,
+                template_id=current_app.config["NEAR_DAILY_LIMIT_TEMPLATE_ID"],  # TODO have a template for sms too
```

Comment:
Following [discussion here](https://gcdigital.slack.com/archives/C0418PT1G6L/p1662476908573219), we just need to update the existing template to show both the sms and email limits in the same email.